### PR TITLE
Updating the README.md about the Build Procedure.

### PR DESCRIPTION
**Pull request by PrernaSingh587**
I have updated the README.md modifying the Build Procedure as mentioned in https://github.com/conda-forge/conda-forge.github.io/blob/master/.ci_scripts/update_docs
This might fix the Issue #1294 

PR Checklist:

- [ ] make all edits to the docs in the `src` directory, not in `docs`
- [x ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x ] put any other relevant information below

### CODE REVIEW

**Comment by PrernaSingh587** [comment]
@viniciusdc @beckermr 
Hi , what do you think of this? Is this going in the right direction?

**Comment by viniciusdc** [comment]
@beckermr I asked her to do these modifications directly to the `README` file (not the one inside `src`) because I don't think the CI manage this one

File: README.md [hunk 1]
```diff
@@ -12,10 +12,16 @@ If you have questions or need help, please check out our documentation for a [li
...
-1. `conda env create -f ./.ci_scripts/environment.yml`
-2. `conda activate conda-forge-docs`
-3. `cd src`
-4. `make html`
+1.  Clone the repository into your local machine.
+2.  Go into the main folder. </br>
+    Run the following commands.  
+3. `conda env create -f ./.ci_scripts/environment.yml`
+4. `conda activate conda-forge-docs`
+5. `pushd newsfeed`
```
Comment (viniciusdc) [inline README.md:20]:
```suggestion
5. `cd newsfeed && pip install --no-deps .`
```

File: README.md [hunk 2]
```diff
@@ -12,10 +12,16 @@ If you have questions or need help, please check out our documentation for a [li
...
+6. `pip install --no-deps .`
+7. `popd`
```
Comment (viniciusdc) [inline README.md:22]:
```suggestion
7. `cd ../src`
```

File: README.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ To build the docs locally, you should do the following:
     Run the following commands.  
 3. `conda env create -f ./.ci_scripts/environment.yml`
 4. `conda activate conda-forge-docs`
-5. `pushd newsfeed`
+5. `cd newsfeed && pip install --no-deps .`
 6. `pip install --no-deps .`
```
Comment (viniciusdc) [inline README.md:21]: you can remove this line now

**Pull request merged**